.github/workflows/run-unit-tests.yml

```diff
@@ -14,6 +14,8 @@ jobs:
   test-on-macos:
     name: Test on macOS
     runs-on: macos-13
+    env:
+      INSTALL_DOCKER: '0'  # Set to '0' to skip Docker installation
```

Comment:
Why it is skipped?

Reply:
Just add it back here:
https://github.com/OpenDevin/OpenDevin/pull/2342